store/tikv/kv.go

```diff
@@ -105,9 +105,10 @@ type tikvStore struct {
 	lockResolver *LockResolver
 	gcWorker     *GCWorker
 	etcdAddrs    []string
+	mock         bool
 }
 
-func newTikvStore(uuid string, pdClient pd.Client, client Client, enableGC bool) (*tikvStore, error) {
+func newTikvStore(uuid string, pdClient pd.Client, client Client, enableGC bool, mock bool) (*tikvStore, error) {
```

Comment:
We don't need to pass `mock`, it can be inferred by asserting the type of the client.